extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -21,22 +21,46 @@ import {
 import {CSS} from '../../../build/amp-story-interactive-slider-0.1.css';
 import {htmlFor} from '#core/dom/static-template';
 
+/**
+ * Generates the template for the slider.
+ *
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildSliderTemplate = (element) => {
+  const html = htmlFor(element);
+  return html`
+    <div class="i-amphtml-story-interactive-slider-container">
+      <div class="i-amphtml-story-interactive-prompt-container"></div>
+      <div class="i-amphtml-story-interactive-slider-range-container">
+        <input
+          class="i-amphtml-story-interactive-slider-range-input"
```

Comment:
Can we change these classes to `i-amphtml-story-interactive-slider-input-container` and `i-amphtml-story-interactive-slider-input`?